plugins/node.d/amavis

```diff
@@ -31,36 +33,34 @@ plugin usually runs at.  To enable log reading it needs to run as a
 group or user that has read access, as shown above.
 
 By default the logtail program is started without any explicit path,
-but if it is not found in the system $PATH then you will need to specify
+but if it is not found in the system $PATH then you'll need to specify
 the full path for the program.
 
 =head2 EXAMPLE CONFIGURATION
 
 The following shows a typical configuration:
 
   [amavis]
-     env.amavislog     /var/log/mail/mail.info
-     env.logtail       /usr/bin/logtail
+     env.amavislog     /var/log/maillog
+     env.logtail       /usr/local/bin/logtail
+     env.amavisd       /usr/local/sbin/amavisd
      group adm
 
-=head2 DEFAULT CONFIGURATION
-
-This is the default configuration:
-
-  [amavis]
-    env.mktemp_command /bin/mktemp
-
 =head1 INTERPRETATION
 
-The plugin shows "probable spam", "surely spam" and "virus".  If your
+The plugin shows "probable spam", "surley spam" and "virus".  If your
 "probable spam" raises you may need to tune your spam configuration to
-classify more spam as "surely spam" and so be able to eliminate it.
+classify more spam as "surley spam" and so be able to elliminate it.
 
 =head1 MAGIC MARKERS
 
   #%# family=auto
   #%# capabilities=autoconf
 
+=head1 VERSION
+
+  $Id$
+
```

Comment:
Starting from this point I am quite convinced, that you copied an old version of the plugin over the one in the repository and simply committed this. This is not a good strategy :(